R/plotly_plotbasic.R

```diff
@@ -22,13 +22,13 @@ plotly_plotbasic <- function(tendril, coloring, term) {
 
   if(coloring %in% c("p", "p.adj", "fish")) {
     plotdata$cc.10 <- pmax(log10(plotdata[[coloring]]), -3)
-    palette_data <- tendril_palette()
+    palette <- tendril_palette()
 
     colorscale = list()
-    for (i in seq_len(length(palette_data$values))) {
-      colorscale[[i]] <- c(palette_data$values[[i]], palette_data$grpalette[[i]])
+    for (i in seq_len(length(palette$values))) {
+      colorscale[[i]] <- c(palette$values[[i]], palette$grpalette[[i]])
     }
-
+    print("Helo")
```

Comment:
What is this?

Reply:
It has been removed already. You probably reviewed it before I committed it.